cmd/ledgerconfig/querycmd/querycmd.go

```diff
@@ -96,5 +107,19 @@ func (c *command) run() error {
 		return err
 	}
 
-	return c.Fprintln(string(config))
+	if string(config) == "null" {
```

Comment:
Why not config == nil ?

Reply:
The server returns a JSON string "null" if there is no config. So I need to compare it to "null".